dagshub/upload/wrapper.py

```diff
@@ -24,6 +27,57 @@ def get_default_branch(src_url, owner, reponame, auth):
     return res.json().get('default_branch')
 
 
+def create_repo(repo_name, is_org=False, org_name="", description="", private=False, auto_init=False, gitignores="",
+                license="", readme="",
+                template="notebook-template"):
+    import dagshub.auth
+    from dagshub.auth.token_auth import HTTPBearerAuth
+
+    token = dagshub.auth.get_token(code_input_timeout=0)
+    if token is not None:
+        auth = HTTPBearerAuth(token)
```

Comment:
Else - what?
There's no way to proceed, and username and password aren't supported.
It should try to take the username, password, and token from `config`, same as the `Repo` object does.

Reply:
@martintali They SHOULD supported - I'm just saying that you should be reusing the `config` package here.
(In the current implementation, they aren't checked, only the token - and if there is no token, the call will always fail, so there's no point in even attempting it)